ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/Constants.java

```diff
@@ -53,7 +53,7 @@
   public static Bytes BLS_WITHDRAWAL_PREFIX;
 
   // Time parameters
-  public static int SECONDS_PER_SLOT = 6;
+  public static int SECONDS_PER_SLOT = 10;
```

Comment:
do u want to keep 10 second slots?  it is working with 6 second slots (without aggregations)